lib/debug/observatory_debugger.dart

```diff
@@ -472,6 +472,9 @@ class ObservatoryIsolate extends DebugIsolate {
 
   String get name => isolateRef.name;
 
+  String get displayName =>
+    name.contains(r'$') ? "${name.replaceAll(r'$', ':')}()" : name;
```

Comment:
Could you add a comment as to why the `replaceAll` is needed?
And why append `()` to the displayName if it contains `$`?